ECHO-COVID19.ipynb

```diff
@@ -56,14 +262,32 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "my_state=\"LA\" # Could switch to dropdown menu"
+    "# Third, let's look at how MUCH these facilities exceeded their permits.\n",
+    "# We'll calculate the median level (in percent terms) permits were exceeded by.\n",
+    "med = exceeds.reset_index()\n",
+    "med = med.set_index(['MONITORING_PERIOD_END_DATE'])\n",
+    "med = med.resample(\"M\").median()\n",
+    "med = med.loc[(med.index.month == 4)]\n",
+    "med\n",
+    "\n",
+    "ax = med.plot(kind='bar', title = \"Past Aprils - MEDIAN % EXCEEDENCE\", figsize=(20, 10), fontsize=16)\n",
+    "ax\n",
+    "\n",
+    "#label trendline\n",
+    "trend=med['EXCEEDENCE_PCT'].mean()\n",
```

Comment:
this misspelling of exceedance is from the dataset, right?

Reply:
I think so!